### PR TITLE
watch-later, history feature added

### PR DESCRIPTION
**Pull request by MonikaJhingan**
Pls review asap!!!!

### CODE REVIEW

File: src/components/VideoHistory/VideoHistory.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        method: "get",
+        url: "/api/user/history",
+        headers: { authorization: auth.token },
+      });
+      setHistory(response.data.history);
+    })();
+  }, [auth.token, setHistory]);
+
+  const removeFromHistory = (_id) => {
+        (async () => {
```
Comment (JantuDeb) [inline src/components/VideoHistory/VideoHistory.js:24]: Don't need to use iife here as you are invoking the function

File: src/components/VideoWatchLater/VideoWatchLater.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    })();
+  }, [auth.token, setWatchLaterVideos]);
+
+  const removeWatchLater = (_id) => {
+    (async () => {
+      const response = await axios.delete(`/api/user/watchlater/${_id}`, {
+        headers: { authorization: auth.token },
+      });
+      setWatchLaterVideos(response.data.watchlater);
+    })();
```
Comment (JantuDeb) [inline src/components/VideoWatchLater/VideoWatchLater.js:29]: same here

**Review by JantuDeb — APPROVED** [review]
Overall the code looks fine. I have left few suggestions you can take a look

**Review by neetukumari4858 — COMMENTED** [review]
ovarall looks fine
good to merge

**Review by neetukumari4858 — APPROVED** [review]

**Pull request merged**